CHANGES.txt

```diff
@@ -1 +1,70 @@
-Please see CHANGES.txt in package
+=======
+Changes
+=======
+
+------------------
+3.0.0 (unreleased)
+------------------
+
+- Package the zcml files.
+
+- Revived from svn.zope.org, removed browser parts, updated dependencies.
```

Comment:
That "removed browser parts" is a backwards-incompatible change that should be given more prominence, like a separate list item in the changelog.

Reply:
done